### PR TITLE
Add cask for bootstrap studio 5.0.3

### PR DESCRIPTION
**Pull request by khrj**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by vladimyr** [comment]
What is the output of:
```
./developer/bin/find-appcast /Applications/Bootstrap\ Studio.app
```

**Comment by KhushrajRathod** [comment]
```bash
$ ./developer/bin/find-appcast /Applications/Bootstrap\ Studio.app
Looking for Sparkle appcast… Not found.
Looking for electron-builder appcast… Not found.

An "app-update.yml" file was found, but we could not parse it.
Please report this on https://github.com/Homebrew/homebrew-cask/
Provide this info:

provider: generic
url: 'https://bootstrapstudio.io/app/updates'
updaterCacheDirName: bstudio-updater
```

**Comment by vladimyr** [comment]
Thanks, sadly that URL does not contain appcast. I'll inspect it manually and let you know 👍

**Comment by KhushrajRathod** [comment]
@vladimyr I don't know if this helps, but the list of old releases is on the same page as the download - 
https://bootstrapstudio.io/download/#accordion

**Comment by vladimyr** [comment]
> @vladimyr I don't know if this helps, but the list of old releases is on the same page as the download - 
> https://bootstrapstudio.io/download/#accordion

We are in search of a machine-readable file (XML, YAML, something along these lines) that gets fetched during app startup. This isn't it but thanks again for helping! 👍

File: Casks/bootstrap-studio.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask 'bootstrap-studio' do
+  version '5.0.3'
+  sha256 'd208d5bb2b87ec1ea65ab2017317f35050a8d13c96e5bf6106817ef47c1fea6f'
+
+  url "https://bootstrapstudio.io/releases/desktop/#{version}/Bootstrap%20Studio.dmg"
```
Comment (vladimyr) [inline Casks/bootstrap-studio.rb:5]:
```suggestion
  url "https://bootstrapstudio.io/releases/desktop/#{version}/Bootstrap%20Studio.dmg"
  appcast "https://bootstrapstudio.io/app/updates/latest-mac.yml"
```
The app is trying to fetch updates from given URL but it results in 404 😩

Comment (KhushrajRathod) [reply Casks/bootstrap-studio.rb:5]: That file has no content

Comment (vladimyr) [reply Casks/bootstrap-studio.rb:5]: I'm aware of that but that is an error on the upstream side. As soon as you run Bootstrap Studio it tries to download it.

Comment (KhushrajRathod) [reply Casks/bootstrap-studio.rb:5]: From the documentation:

> An appcast can be any URL hosted by the app’s developer that changes every time a new release is out or that contains the version number of the current release (e.g. a download HTML page). Webpages that only change on new version releases are preferred, as are sites that do not contain previous version strings (i.e. avoid changelog pages if the download page contains the current version number but not older ones). (Example: razorsql.rb)

The download page, https://bootstrapstudio.io/download, changes every time a release is published. Maybe that would be better? Or do you think I should just merge https://bootstrapstudio.io/app/updates/latest-mac.yml as you suggested earlier?

Comment (KhushrajRathod) [reply Casks/bootstrap-studio.rb:5]: Even https://bootstrapstudio.io/pages/releases changes

Comment (vladimyr) [reply Casks/bootstrap-studio.rb:5]: Personally I would go with https://bootstrapstudio.io/pages/releases but that is question for @vitorgalvao

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**